testing/test_session.py

```diff
@@ -253,3 +256,34 @@ def pytest_sessionfinish():
     """)
     res = testdir.runpytest("--collect-only")
     assert res.ret == EXIT_NOTESTSCOLLECTED
+
+
+@pytest.mark.parametrize("path", ["root", "{relative}/root", "{environment}/root"])
+def test_rootdir_option_arg(testdir, path):
+    if 'relative' in path:
+        path = path.format(relative=os.getcwd())
```

Comment:
You should not be using `os.getcwd()` here because this might leave some files after the run (the `.pytest_cache` directory for example). Use `str(testdir.tmpdir)` instead.